Update seer docs (#14027)
getsentry/sentry-docs: docs/product/ai-in-sentry/seer/index.mdx

sidebar_order: 21
---

Seer is Sentry's AI debugging agent. It uses Sentry's rich context (issue details, tracing data, logs, and profiles) to help you troubleshoot and fix errors and performance issues faster.

## Seer Capabilities

Seer provides three main capabilities to enhance your debugging:

- [**Issue fix**](#issue-fix): A collaborative agent skilled at root cause analysis and solving issues
- [**Issue scan**](#issue-scan): Automatically scans issues as they come in, feeding additional context to your alerts and automating triage

> - [**Automation**](#automation): Combine these tools into powerful workflow: automatically identifying actionable issues in real time with scanner, and then generating high quality fixes

Reviewer: Do we think the reader will know what 'scanner' is in this context?

> - [**Automation**](#automation): Combine these tools into powerful workflow: automatically identifying actionable issues in real time with scanner, and then generating high quality fixes

Reviewer: @ItalyPaleAle That looks good, I would still put an 'a' before 'powerful'

> - [**Automation**](#automation): Combine these tools into powerful workflow: automatically identifying actionable issues in real time with scanner, and then generating high quality fixes

Reviewer 2: ```suggestion
- [**Automation**](#automation): Combines issue fix and scan into a powerful workflow: identifying actionable issues in real time and generating high quality fixes, automatically
```

### Issue Fix

Seer combines all of the relevant context from your code with Sentry's telemetry data to provide a best-in-class issue debugging experience. The workflow consists of three steps:

1. Root cause the issue using Sentry's context and your codebase
2. Identify a solution based on the analysis
3. Generate a code patch that implements the solution and open a PR

For a full product walk-through, please visit [issue fix](./issue-fix/).

<VimeoEmbed id="1091964967" />

### Issue Scan

Seer is able to automatically analyze issues as they are ingested by Sentry. This initial review is used to:

- Generate an "actionability" score, which indicates how likely it is that a particular issue can be fixed with a code change
- Augment issue alerts with a helpful analysis about the problem
- Add an initial guess for what the potential problem is to the issue details page

> The actionability score is represented in the issue feed by a Seer icon, which indicate issues that are likely straightforward fixes.

Reviewer: A screenshot or arcade here might help illustrate this.
